network-libp2p/src/connection_pool/behaviour.rs

```diff
@@ -29,23 +29,37 @@ use wasm_timer::Interval;
 use super::Error;
 use crate::discovery::peer_contacts::PeerContactBook;
 
+/// Current state of connections and peers for connection limits
 #[derive(Clone, Debug)]
 struct Limits {
+    /// Number of peers connected per IP
     ip_count: HashMap<IpAddr, usize>,
+    /// Number of peers connected per IP subnet
     ip_subnet_count: HashMap<IpNetwork, usize>,
+    /// Total peer count
     peer_count: usize,
 }
 
+/// Connection pool behaviour configuration
 #[derive(Clone, Debug)]
 struct Config {
+    /// Desires count of peers
     peer_count_desired: usize,
+    /// Maximum count of peers
     peer_count_max: usize,
+    /// Maximum peer count per IP
     peer_count_per_ip_max: usize,
+    /// Maximum peer count per subnet
     peer_count_per_subnet_max: usize,
+    /// IPv4 subnet mask to apply to detect same connections for the same subnet
     ipv4_subnet_mask: u8,
```

Comment:
```suggestion
    ipv4_subnet_prefix_len: u8,
```
It's not really a mask.